backend/src/main/java/capstoneexpensetracker/backend/travelergroup/TravelerGroupService.java

```diff
@@ -38,4 +39,12 @@ public void deleteTravelerGroup(String id) {
         }
         travelerGroupRepository.deleteById(id);
     }
+
+    public TravelerGroup updateTravelerGroupById(String id, TravelerGroup travelerGroup) {
+        Optional<TravelerGroup> travelerGroupToFind = travelerGroupRepository.findById(id);
+        if (travelerGroupToFind.isPresent()) {
+            return travelerGroupRepository.save(travelerGroup);
```

Comment:
Your repository has an implicit method existsById() which you could also use here